aws/cloudformation/components/codeprojects.yml.erb

```diff
@@ -0,0 +1,68 @@
+<%# Note this file must be indented one level, as it is a child of 'Resources'. %>
+  # -----------------------------------------------
+  # Begin codeprojects.org resources
+  # 
+  # This template component includes resources for the codeprojects.org
+  # infrastructure. Currently this is mostly manually deployed.
+  # 
+  # -----------------------------------------------
+
+<%-
+  hosted_zone_id = "Z2E5MLW9OSPK9O"
```

Comment:
Can we make the Hosted Zone ID for the base domain name a template Parameter? I’m thinking ahead to when we try to make our systems deployable to any AWS Account.

Reply:
Rather than a parameter, we can fetch it using the AWS client library. Updated.

Reply:
Awesome! At some point, I’m interested in making the base domain a Parameter as well so that in, say, our Dev AWS account its (`dev-codeprojects.org`).